### PR TITLE
add `<<`, the unindenting function

### PR DESCRIPTION
**Pull request by atstp**
The function should match everything mentioned in #24, and there's a test for each arity :heavy_check_mark: 

Should anything special be added for the docs?

### CODE REVIEW

File: src/cuerdas/core.cljc [hunk 1]
```diff
@@ -676,3 +676,17 @@
...
             (split suffix)
             first)))
+
+(defn <<
+  "Unindent multiline text.
+  Uses either a supplied regex or the shortest
+  beginning-of-line to non-whitespace distance"
+  ([r s] (->> s lines (map #(replace % r "")) unlines))
+  ([s]
+   (let [all-indents (->> (rest (lines s)) ;; ignore the first line
```
Comment (niwinz) [inline src/cuerdas/core.cljc]: Please, can you sort the arities in the ascendant order, `(fn ([x] ...) ([x y] ...) ([x y z] ...))`

**Comment by niwinz** [comment]
Everything is fine, would be awesome have a little section on docs for this function but is not stoper for merge this PR. 

Please, fix the commented issue and I'll merge it ASAP.

Thanks!

**Comment by atstp** [comment]
whoops, just forgot to push it.

File: src/cuerdas/core.cljc (hunk 1 above)
Comment (atstp) [inline src/cuerdas/core.cljc]: yep, it's swapped and rebased

**Comment by niwinz** [comment]
Thanks!

**Comment by niwinz** [comment]
and `1.0.0-SNAPSHOT` released on clojars

**Pull request merged**